fetch.bs

```diff
@@ -5395,12 +5392,12 @@ run these steps:
     <ol>
      <li>
       <p>If the user agent is not configured to block cookies for <var>httpRequest</var> (see
-      <a href=https://datatracker.ietf.org/doc/html/rfc6265#section-7>section 7</a> of
+      <a href=https://httpwg.org/specs/rfc6265.html#privacy-considerations>section 7</a> of
       [[!COOKIES]]), then:
 
       <ol>
        <li><p>Let <var>cookies</var> be the result of running the "cookie-string" algorithm (see
-       <a href=https://datatracker.ietf.org/doc/html/rfc6265#section-5.4>section 5.4</a> of
+       <a href=https://httpwg.org/specs/rfc6265.html#cookie>section 5.4</a> of
```

Comment:
Maybe use a section name instead of number while you're here. Or change this to the cross-reference style below you used for chapters?

Same for other cookie spec updates as below.

Reply:
I'm not going to touch these as cookies still need a general overhaul to account for cross-site cookie blocking, `SameSite`, etc.